docs/core/project-sdk/msbuild-props.md

```diff
@@ -1421,6 +1421,116 @@ The `RunWorkingDirectory` property defines the working directory for the applica
+
+### EnableAspireTesting
+
+When you use the [MSTest project SDK](../testing/unit-testing-mstest-sdk.md), you can use the `EnableAspireTesting` property to bring in all the dependencies and default `using` directives you need for testing with `Aspire` and `MSTest`.
+
+For more information, see [Test with .NET Aspire](../testing/unit-testing-mstest-sdk.md#net-aspire).
+
+### EnablePlaywright
+
+When you use the [MSTest project SDK](../testing/unit-testing-mstest-sdk.md), you can use the `EnablePlaywright` property to bring in all the dependencies and default `using` directives you need for testing with `Playwright` and `MSTest`.
+
+For more information, see [Playwright](../testing/unit-testing-mstest-sdk.md#playwright).
+
+### EnableMSTestRunner
+
+The `EnableMSTestRunner` property enables or disables the use of the [MSTest runner](../testing/unit-testing-mstest-runner-intro.md). The MSTest runner is a lightweight and portable alternative to VSTest.
+
+> [!NOTE]
+> If your project specifies the [MSTest SDK](../testing/unit-testing-mstest-sdk.md), you don't need to set this property. It's set automatically.
+
+### EnableNUnitRunner
+
+The `EnableNUnitRunner` property enables or disables the use of the [NUnit runner](../testing/unit-testing-nunit-runner-intro.md). The NUnit runner is a lightweight and portable alternative to VSTest.
+
+### GenerateTestingPlatformEntryPoint
+
+Setting the `GenerateTestingPlatformEntryPoint` property to `false` disables the automatic generation of the program entry point in a test project. You might want to set this property to `false` when you manually define an entry point, or when you reference a test project from an executable that also has an entry point.
+
+For more information, see [error CS8892](../testing/unit-testing-platform-faq.md#error-cs8892-method-testingplatformentrypointmainstring-will-not-be-used-as-an-entry-point-because-a-synchronous-entry-point-programmainstring-was-found).
+
+### TestingPlatformCaptureOutput
+
+The `TestingPlatformCaptureOutput` property controls whether all console output that a test executable writes is captured and hidden from the user when you use `dotnet test` to run `Microsoft.Testing.Platform` tests. By default, the console output is hidden. This output includes the banner, version information, and formatted test information. Set this property to `false` to show this information together with MSBuild output.
+
+For more information, see [Show complete platform output](../testing/unit-testing-platform-integration-dotnet-test.md#show-complete-platform-output).
+
+### TestingPlatformCommandLineArguments
+
+The `TestingPlatformCaptureOutput` property lets you specify command-line arguments to the test app when you use `dotnet test` to run `Microsoft.Testing.Platform` tests. The following project file snippet shows an example.
+
+```xml
+<PropertyGroup>
+  ...
+  <TestingPlatformCommandLineArguments>--minimum-expected-tests 10</TestingPlatformCommandLineArguments>
```

Comment:
Is this syntax correct for the value?

Reply:
Not seeing a lot of examples, but it does look correct based on these findings: https://grep.app/search?q=TestingPlatformCommandLineArguments

Reply:
It's a normal msbuild prop that we concat, because parameters are "open" https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-platform-integration-dotnet-test#dotnet-test---microsofttestingplatform-mode